### DISCUSSION TITLE
src="//:0" gives `/:0: ENOENT: no such file or directory, open '/:0'`

### DISCUSSION THREAD

**Discussion by Philzen** [parcel-bundler/parcel#4760] [General] [answered]
# 🐛 bug report

To keep html validity and also [avoid some browsers making unneccessary requests](https://stackoverflow.com/questions/19126185/setting-an-image-src-to-empty), i am setting `src="//:0"` on all resources that are lazy loaded in the project. It looks as if parcel falls over then with `🚨  /:0: ENOENT: no such file or directory, open '/:0'`

## 🎛 Configuration (.babelrc, package.json, cli command)

```js
{
  "name": "stadtwerk-mit-zukunft",
  "description": "",
  "version": "1.0.0",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/B-FfF/stadtwerk-mit-zukunft.git"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/B-FfF/stadtwerk-mit-zukunft/issues"
  },
  "homepage": "https://github.com/B-FfF/stadtwerk-mit-zukunft#readme"
}
```

## 🤔 Expected Behavior

I guess parcel should just build ;) Evaluating parcel for the first time and trying to throw it at an existing project.  However i can never load the dev server page.

## 😯 Current Behavior

Error `🚨  /:0: ENOENT: no such file or directory, open '/:0'` is shown, cpu load is high steadily and it seems as if parcel is stuck in a loop, as continuously the error message from #3643 pops up and disappears again in terminal. Not sure if all those behaviours are causally connected.

## 💁 Possible Solution

 ¯\_(ツ)_/¯

## 🔦 Context

Trying to run parcel for the first time on [this project](https://github.com/B-FfF/stadtwerk-mit-zukunft/tree/gh-pages).

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.12.4
| Node             | v10.21.0
| npm/Yarn         | 6.14.4
| Operating System | Linux 5.4.44-1-MANJARO

**Comment by DeMoorJasper** [comment]
Seems like our url check might be flawed, these should get detected as a url and be skipped entirely.

Does this still happen with Parcel 2? `yarn add parcel@next` and make sure parcel-bundler is no longer installed to prevent duplicate cli binaries

**Comment by DeMoorJasper** [comment, marked as answer]
Just tested it and works in Parcel 2

**Comment by Philzen** [comment]
Thanks for taking the time to look into this!

I can confirm this error does not come up anymore in 2.0.0-beta.1
